tests/ga/test_multi_config_extension.py

```diff
@@ -738,7 +738,7 @@ def __assert_env_variables(handler_name, handler_version="1.0.0", seq_no="1", ex
                     self.assertFalse(any(env_var in commands['data'] for env_var in not_expected), "Unwanted env variable found")
 
         def mock_popen(cmd, *_, **kwargs):
-            if 'env' in kwargs:
+            if 'env' in kwargs and ExtCommandEnvVariable.ExtensionVersion in kwargs['env']:
```

Comment:
we should add a unit test for the environment variable, or modify the existing tests to check that the var is in the env of the test processes

Reply:
added